### PR TITLE
deprecate: mode

### PR DESCRIPTION
**Pull request by jacoobes**
# Deciding to remove experimental mode for sern/handler npm package
- the handler package shouldn't have this type of logic. This should be up to the CLI build tool or user.
- For random debug statements, i'll probably use esbuild's drop label functionality
- It tightly couples logic to data and it isn't the best solution.

### CODE REVIEW

File: src/types/core.ts [hunk 1]
```diff
@@ -9,9 +9,9 @@ export interface Wrapper {
     defaultPrefix?: string;
     events?: string;
     /**
-     * Overload to enable mode in case developer does not use a .env file.
```
Comment (EvolutionX-10) [inline src/types/core.ts:12]: why did you remove the description?

Comment (jacoobes) [reply src/types/core.ts:12]: nice catch, need to re add

**Review by EvolutionX-10 — APPROVED** [review]
